Automattic/jetpack
# Restore phpcs-changed checks

Follow-up to #13920.

This updates phpcs-changed to the beta version provided by @sirbrillig (thanks!), and refactors the way it works. This results in a much simplified approach.

#### Changes proposed in this Pull Request:
* Upgrade to latest phpcs-changed and use the new automated git workflow.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* p1HpG7-7SV-p2

#### Testing instructions:
* On this branch, make edits to php files both on and off the whitelist. Some edits should be PHPCS compliant. Others not.
* My favorite quick compliant or not is ` // This is a good edit.` and ` // This is a bad edit` (without the trailing period.)

#### Proposed changelog entry for your changes:
* n/a, dev tools only.
